src/app/application/components/route-view/RouteTitle.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <template
+    v-if="false"
```

Comment:
Ah, ok glad you asked, so this has a nit of an explanation. I'm kinda unsure of where we are going from a design perspective and that's why I've left this in and like this. More detail:

---

Quite often in web application pages the HTML `<title>` of the page is the same as the `header/<h*>` of the page, and it can be tedious to specify it twice. Therefore 'ideally' this component would also print the title by default, i.e.

```vue
<h2>
  <RouteTitle 
    :title="Services"
  />
</h2>
```
would render:

```html
<title>Services | Kuma Manager</title>
...
<h2>Services</h2>
```

The RouteTitle component would also have a `:render="false"` (or similarly named) to disable the default behaviour of also rendering the title for the least common case where you don't want to render.

In our application we don't currently render the title as a page header very consistently, sometimes we do sometimes we don't and its around 50/50 I think.

I think there are designs/decisions to make this consistent through the app/the apps, therefore probably using the header/h* of the page as the title most of the time. This is why for the moment its `false`/unused as I'm kinda sure I can just remove this later, or make it so we can `:render="true"` if its the case that rendering is the least common thing I want to do.

Reply:
I see. That makes some sense to me and I would not mind having such titles in the UI in the form of h1. Not super keen on the current state of things either (e.g. the h1s inside the detail views).

Reply:
Well that would mean we would get into a discussion over whether `Kuma Manager` should be h1 😆 , I'm sure we'll get there at some point.